### PR TITLE
Add Nuxt 3 Support :sparkles:

### PR DESCRIPTION
**Pull request by selemondev**

### CODE REVIEW

File: src/cli/config.ts [hunk 1]
```diff
@@ -26,6 +35,18 @@ export interface Plugin {
   addConfigImport: boolean;
 }
 
+
+/**
+ * The '@nuxtjs/tailwindcss' module helps you set up Tailwind CSS (version 3) in your Nuxt 3 application in seconds.
+ */
+export const supportedNuxtModule: readonly Module[] = [
```
Comment (AndrejJurkin) [inline src/cli/config.ts:42]: This NuxtTailwind module seems to work out of the box, let's not ask users about it.

They select Nuxt with this Module.

So please, let's remove this and all related code and simply add the dependency into the Config.

File: src/cli/config.ts [hunk 2]
```diff
@@ -111,6 +135,9 @@ export interface UserInput {
   // Additional dependencies to install specified by the user.
   dependencies: Dependencies[];
 
+  // The Nuxt module(s) to install specified by the user
+  modules: NuxtModule[]
```
Comment (AndrejJurkin) [inline src/cli/config.ts:139]: Don't forget to remove this as well.

File: src/cli/config.ts [hunk 3]
```diff
@@ -185,6 +213,59 @@ export const NEXTJS_TS_CONFIG: AppConfig = {
...
+      path.join(NUXTJS_CONFIG.templateDir, "nuxt.config.js"),
+      path.join(projectDir, "nuxt.config.js"),
+    );
+  },
+  getCssOutputPath: ({ projectDir }) => {
+    return path.join(projectDir, "assets", "main.css");
+  },
+  createInstallCommand: createNuxtCommand,
+  deleteFiles: async ({ projectDir }) => {
+    await fs.remove(path.join(projectDir, "nuxt.config.ts"));
```
Comment (AndrejJurkin) [inline src/cli/config.ts:239]: Why do we need to remove nuxt.config.ts here?

**Review by AndrejJurkin — CHANGES_REQUESTED** [review]
Thank you so much for this PR. It looks good!

But as I want to make this CLI as minimal as possible, I've suggested we should use NuxtTailwind as default option when setting up the project.

So please, remove all code relevant to the Nuxt modules part.

File: src/cli/config.ts (hunk 1 above)
Comment (selemondev) [reply src/cli/config.ts:42]: Okay, I will remove it ✨

File: src/cli/config.ts (hunk 2 above)
Comment (selemondev) [reply src/cli/config.ts:139]: Ok

File: src/cli/config.ts (hunk 3 above)
Comment (selemondev) [reply src/cli/config.ts:239]: To allow the usage of JavaScript. Instead of nuxt.config.ts we will utilise nuxt.config.js

**Pull request closed without merging**